source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

```diff
@@ -18,8 +18,13 @@ This may be fine and the emails will eventually go out, but it's worth some inve
 * `warning` - `content_changes` unprocessed for over 90 minutes
 * `critical` - `content_changes` unprocessed for over 120 minutes
 
-See the [ProcessContentChangeAndGenerateEmailsWorker][content-change-worker] for
-more information.
+Every 30 minutes, [RecoverLostJobsWorker][recover-lost-jobs-worker] automatically requeues jobs for any
+unprocessed content changes and messages that are over 1-hour old.
+
+To initiate recovery immediately, run this rake task manually: [recovery:recover_lost_jobs][].
+
+See the [ProcessContentChangeWorker][content-change-worker]
```

Comment:
Dropping people straight into the worker is quite "heavy".

How about linking to [here](https://docs.publishing.service.gov.uk/manual/email-notifications-how-they-work.html) and (also) expanding those docs with details about these workers?

Reply:
Good idea. I could also add a section in here: https://docs.publishing.service.gov.uk/apis/email-alert-api/troubleshooting.html.

Reply:
https://github.com/alphagov/email-alert-api/pull/1318/files#diff-2ff67fee0173608876ccc012bf3ff6ceR81

Reply:
https://docs.publishing.service.gov.uk/manual/email-notifications-how-they-work.html is high-level presentation information about the app, so I thought I'd link directly to https://docs.publishing.service.gov.uk/apis/email-alert-api/troubleshooting.html.